grafana.yaml

```diff
@@ -0,0 +1,78 @@
+package:
+  name: grafana
+  version: 10.2.0
+  epoch: 0
+  description: The open and composable observability and data visualization platform.
+  copyright:
+    - license: AGPL-3.0-or-later
+  dependencies:
+    runtime:
+      - bash
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - git
+      - go
+      - yarn
+      - posix-libc-utils
+      - nodejs-16
+      - findutils
+      - py3-setuptools
+      - python3
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/grafana/grafana
+      tag: v${{package.version}}
+      expected-commit: ae830f687450b8a0aca94ab2d72cc08853a80fff
+
+  - runs: |
+      # Mititage CVE-2022-41723 / GHSA-vvpx-j8f3-3w6h
+      # Mitigate CVE-2023-39325, CVE-2023-3978, CVE-2023-44487
+      go mod edit -replace golang.org/x/net=golang.org/x/net@v0.17.0
+      go mod edit -replace go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp=go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp@v0.44.0
+      go get go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace@v0.44.0
+      go mod tidy
+
+      yarn install
+      yarn run build
+      yarn run plugins:build-bundled
+
+      # Our default LDFLAGS cause an issue with grafana. We need to remove '-z, noexecheap' from LDFLAGS
```

Comment:
should we create an issue tracking what the issue is?

Reply:
or document what the issue is so if later it gets fixed we can remove this.

Reply:
What issue does it cause?  In general, programs should not crash with this flag, as they should not be executing code from memory that is on the heap.

Reply:
Filed an issue: https://github.com/wolfi-dev/os/issues/7419